packages/environment-check/src/checks/environment.ts

```diff
@@ -137,26 +219,32 @@ async function getDestinationsResults(
  */
 export async function checkEnvironment(options?: CheckEnvironmentOptions): Promise<EnvironmentCheckResult> {
     const logger = getLogger();
+    let destinations: Destination[];
+    let destinationResults: { [dest: string]: DestinationResults };
 
     const { environment, messages } = await getEnvironment();
```

Comment:
we now need to set the ``requestedChecks` otherwise they are not part of the markdown if you run it from command line:

```suggestion
    const requestedChecks: Set<Check> = new Set();
    
    requestedChecks.add(Check.Environment);
    const { environment, messages } = await getEnvironment();
```

Reply:
Thanks Klaus, slipped my mind that we could generate the markdown from the CLI. Updated now